docs/reference/operators/mobile_package_op_type_support_1.9.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: ORT 1.9 Mobile Package Operator and Type Support
-parent: Pre-Built Package
-grand_parent: ONNX Runtime Mobile
-nav_exclude: true
+title: ORT 1.9 Mobile Package Operators
+parent: Operators
```

Comment:
Should we add 1.10 while we're at it? The release branch is cut so the operators set doco can be generated on it.

Reply:
Yes, we should! Is this auto-generated? Where would I find it?

Reply:
In the ORT enlistment /tools/python/gen_ort_mobile_pkg_doc.py will update /docs/ORTMobilePackageOperatorTypeSupport.md

Reply:
Updated version is also checked in to master so you can copy from /docs/ORTMobilePackageOperatorTypeSupport.md

Reply:
Added